larsim/ElectronDrift/DriftElectronstoPlane_module.cc

```diff
@@ -294,6 +294,11 @@ namespace detsim {
         posOffsetxyz[0] = posOffsets.X();
         posOffsetxyz[1] = posOffsets.Y();
         posOffsetxyz[2] = posOffsets.Z();
+	if (posOffsetxyz[0] < -1E9 || posOffsetxyz[0] > 1E9 ||
+	    posOffsetxyz[1] < -1E9 || posOffsetxyz[1] > 1E9 ||
+	    posOffsetxyz[2] < -1E9 || posOffsetxyz[2] > 1E9) {
+	  continue;
+	}
```

Comment:
Using utility function:

```.cpp
if (out_of_bounds(posOffsetxyz)) {
  continue;
}
```

Reply:
Hi Kyle -- Thanks much.  I added a new utility function in larsim/Utils/SCEOffsetBounds.cxx,.h and called it from the four places I had previously hardcoded the bounds check.  I updated the CMakeLists.txt files to link with the larsim_Utils lib where necessary.